bin/setup

```diff
@@ -10,8 +10,12 @@ source "${CURRENT_PATH}/support/sh/functions/cic_docker.sh"
 
 if ! which docker
 then
-    say "$(ok "Installing docker")"
-    bash "${CURRENT_PATH}/support/sh/install_docker_ubuntu.sh"
```

Comment:
If we're not using this script anymore then we should remove the file as well.... or are we planning on telling the users to run the script?

Reply:
I did think about removing it but wondered if we wanted to provide it for those that are using using a compatible distro and want a shortcut? 

It might end up doing more harm than good from a maintenance point of view. We could remove it for now and see what feedback we get?

Reply:
Let's leave it in place, but call out that it exists when we print the message saying that the user needs to install it themselves. That way it justifies leaving it in the repo, and it might actually come in helpful for people.

Reply:
How do you want the message to read? E.g.
> If you're using debian based linux O/S try running `bin/support/sh/install_docker_ubuntu.sh`.

It might be just as well to say the following?: 
> Got to https://docs.docker.com/install/ and follow the instructions for your O/S

Reply:
Let's say both of those things :-)